product_docs/docs/pgd/5/consistency/column-level-conflicts.mdx

```diff
@@ -121,7 +121,7 @@ It can, however, have various unexpected effects:
 !!! Note
     Statement and transaction timestamps might be added in the future, which will address issues with mixing effects of concurrent statements or transactions. Still, neither of these options can ever produce results equivalent to commit order.
 
-You can also use the actual commit timestamp, although this feature is considered experimental. To use the commit timestamp, set the last parameter to `true` when enabling column-level conflict resolution:
+You can also use the actual commit timestamp, although this feature is considered experimental. Use of the commit timestamp can only be specified via the deprecated function `bdr.column_timestamps_enable`. Set the last parameter to `true` when enabling column-level conflict resolution:
```

Comment:
Do we really want to say experimental when it relies on a feature now deprecated and not removed through two major versions?

Reply:
@ibarwick any update on this?

Reply:
Update following internal clarification, apologies for the delay.